### PR TITLE
Feature/save cover to array

### PR DESCRIPTION
**Pull request by mdflynn**
## What is the change?
This adds the ability to save generated covers to the saved covers page and then display the covers.

 ## Is this a fix or a feature? 
This is a feature.

 ## Where should the reviewer start? 
Lines 108-128 in the main.js file

## How should this be tested? 
Open index.html in the terminal. Save random covers and created covers, then navigate to the saved cover page to see them displayed.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -106,6 +105,27 @@ function clickHandler(event) {
   homeFromUserPage();
 }
 
+function saveCoverToArray() {
```
Comment (coopterrones) [inline src/main.js:108]: pushing to array

File: src/main.js [hunk 2]
```diff
@@ -106,6 +105,27 @@ function clickHandler(event) {
...
+  if (!savedCovers.includes(currentCover)) {
+    savedCovers.push(currentCover);
+  }
+}
+
+function displaySavedCover() {
+  var coverData = '';
+  for (var i = 0; i < savedCovers.length; i++) {
```
Comment (coopterrones) [inline src/main.js:116]: iteration through the data

File: src/main.js [hunk 3]
```diff
@@ -106,6 +105,27 @@ function clickHandler(event) {
...
+    var addDom = `
+      <div class="mini-cover" id="${savedCovers[i].id}">
```
Comment (coopterrones) [inline src/main.js:118]: interpolation to add elements to the html

**Review by coopterrones — APPROVED** [review]

File: src/main.js [hunk 4]
```diff
@@ -106,6 +105,27 @@ function clickHandler(event) {
...
+        <img class="cover-image" src="${savedCovers[i].cover}"/>
+        <h2 class="cover-title">${savedCovers[i].title}</h2>
+        <h3 class="tagline">A tale of <span>${savedCovers[i].tagline1}</span> and <span>${savedCovers[i].tagline2}</span></h3>
+      </div>
+    `;
+    coverData += addDom;
+  }
+  savedCoverSection.innerHTML = coverData;
+}
```
Comment (Ish451) [inline src/main.js:127]: Nice work figuring out the bugs with the mini covers!

**Review by Ish451 — APPROVED** [review]
Nice work figuring out the bugs with the mini covers!

**Pull request merged**